actions/generate-release-notes/generate-release-notes.sh

```
@@ -19,11 +19,10 @@ fetch_merged_prs() {
     local repo_name="$1"
     local branch="$2"
     local start_datetime="${3}" # Full ISO timestamp: YYYY-MM-DDTHH:MM:SSZ
-    local start_date="${3%%T*}" # Extract YYYY-MM-DD from ISO timestamp
-    
+
     echo "[INFO] Fetching PRs for ${repo_name} (branch: ${branch}) since ${start_datetime}..." >&2
 
-    local prs=$(gh pr list --state merged --base "$branch" --search "merged:>=$start_date" --json number,title,author,mergedAt | jq -r -c --arg start "$start_datetime" '.[] | select(.mergedAt >= $start) | @base64')
+    local prs=$(gh pr list --state merged --base "$branch" --limit 1000 --json number,title,author,mergedAt | jq -r -c --arg start "$start_datetime" '.[] | select(.mergedAt >= $start) | @base64')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)